gimme

```diff
@@ -286,6 +287,20 @@ _wipe_version() {
 	fi
 }
 
+_list_versions() {
+	(
+		if [ ! -d "${GIMME_VERSION_PREFIX}" ] ; then
+			return 0
+		fi
+
+		cd "${GIMME_VERSION_PREFIX}"
+		for d in $(ls -1d *) ; do
```

Comment:
At this point, it might be worth setting `nullglob` up by `set -e`...  :innocent: